drivers/storage/portworx/component/csi.go

```diff
@@ -72,14 +72,18 @@ func (c *csi) Initialize(
 }
 
 func (c *csi) IsEnabled(cluster *corev1.StorageCluster) bool {
-	return pxutil.FeatureCSI.IsEnabled(cluster.Spec.FeatureGates)
+	return pxutil.IsCSIEnabled(cluster)
 }
 
 func (c *csi) Reconcile(cluster *corev1.StorageCluster) error {
 	ownerRef := metav1.NewControllerRef(cluster, pxutil.StorageClusterKind())
 	pxVersion := pxutil.GetPortworxVersion(cluster)
 	csiConfig := c.getCSIConfiguration(cluster, pxVersion)
 
+	if cluster.Status.DesiredImages == nil {
```

Comment:
This normally won't be called, as we usually set defaults in another function. However, for tests, this is needed.